docs/source/guides/data.rst

```diff
@@ -277,17 +282,17 @@ When using PackedSequence, do 2 things:
         return x, y
 
 
-    # In module
-    def training_step(self, batch, batch_nb):
+    # In LightningModule
+    def training_step(self, batch, batch_idx):
         x = rnn.pack_sequence(batch[0], enforce_sorted=False)
         y = rnn.pack_sequence(batch[1], enforce_sorted=False)
 
-----------
 
 Truncated Backpropagation Through Time (TBPTT)
 ==============================================
+
 There are times when multiple backwards passes are needed for each batch.
-For example, it may save memory to use Truncated Backpropagation Through Time when training RNNs.
+For example, it may save memory to use ``Truncated Backpropagation Through Time`` when training RNNs.
```

Comment:
```suggestion
For example, it may save memory to use **Truncated Backpropagation Through Time** when training RNNs.
```
Was the intention to make it bold?

Reply:
nah! but it's better.